src/components/Button/Button.scss

```diff
@@ -29,10 +29,25 @@
   }
 }
 
+.bnt-transparent {
+  border: 1.5px solid $red;
+  color: $red;
+  background: transparent;  
+  @include text(20px, 500, 24px);
+
+  &:hover {
+    opacity: 0.5; 
+  }
+}
+
 .btn-big {
   padding: 16px 74px;
 }
 
+.btn-middle{
```

Comment:
Please add new variants only in case if a new button has different height, don't take into account a width, change width just with additional class or using parent in proper place that it needs